lib/run-in-terminal.coffee

```diff
@@ -180,37 +180,55 @@ module.exports =
                 selectors.tree,
                 "tree-start-terminal-here",
                 "Start terminal here [tree]",
-                () => @tree_start_terminal_here(false)
+                () => @tree_start_terminal_here(false, false)
             ],
             [
                 selectors.tree,
                 "tree-start-terminal-here-and-run",
                 "Start terminal here and run [tree]",
-                () => @tree_start_terminal_here(true)
+                () => @tree_start_terminal_here(true, false)
+            ],
+            [
+                selectors.tree,
+                "tree-start-terminal-here-and-run-with-par",
+                "Start terminal here and run [tree] (with par)",
```

Comment:
`with par` here and below should be replaced with `with parameters`, since this is what user see and this should definetly be maximum readable.